packages/react/src/helpers/utils.ts

```diff
@@ -1 +1,14 @@
 export const isDevelopment = () => process.env.NODE_ENV !== 'production';
+
+export const isInputOrSelectElement = (
+  target: unknown
+): target is HTMLInputElement | HTMLSelectElement => {
+  return (
+    (target as HTMLInputElement)?.nodeName === 'INPUT' ||
+    (target as HTMLSelectElement)?.nodeName === 'SELECT'
+  );
+};
+
+export const isInputElement = (target: unknown): target is HTMLInputElement => {
+  return (target as HTMLInputElement)?.nodeName === 'INPUT';
+};
```

Comment:
Is it worthwhile to break out both input and select checks into separate utils? Could we use them in the future?

```typescript
export const isInputElement = (target: unknown): target is HTMLInputElement =>
  (target as HTMLInputElement)?.nodeName === 'INPUT';

export const isSelectElement = (target: unknown): target is HTMLSelectElement =>
  (target as HTMLSelectElement)?.nodeName === 'SELECT';

export const isInputOrSelectElement = overSome([isInputElement, isSelectElement]);
```

Reply:
Split them into separate functions